Sources/CorePayments/Networking/GraphQL/UpdateVaultSetUpTokenMutation.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+
+public struct PaymentSourceInput: Codable {
+    
+    let card: VaultCard
```

Comment:
I thought for future, we can make this 
public enum PaymentSourceInput: Coddle {
     case card(Card)
     case otherPaymentMethod(OtherPaymentMethod)
}

Reply:
I am working on separating out these structs into separate files. 
Trying to reduce duplication of Card struct.